mobile/native/src/ln_dlc/mod.rs

```diff
@@ -392,15 +395,44 @@ pub fn run(seed_dir: String, runtime: &Runtime) -> Result<()> {
 
         runtime.spawn(track_channel_status(node.clone()));
 
-        if let Err(e) = node.sync_position_with_subchannel_state().await {
-            tracing::error!("Failed to sync position with subchannel state. Error: {e:#}");
-        }
+        runtime.spawn({
```

Comment:
🤔 Could that be potentially dangerous as it's not blocking?

What if the user is opening / closing a position in the meantime?